src/Components/RecentlyViewed.tsx

```diff
@@ -59,6 +59,7 @@ export class RecentlyViewed extends React.Component<RecentlyViewedProps> {
                     return (
                       <FillwidthItem
                         lazyLoad={true}
+                        // @ts-ignore // TODO: Correct typing
```

Comment:
Not 100% sure why it's not picking up the nested fragment type here. But better than the `any` for the time being.